### ISSUE TITLE
[Feature Request] Unlock with PIN

### ISSUE THREAD

**Issue by creeper451** [hardcore-sushi/DroidFS#188]
It's kinda hard to explain but bear with me please🙏, so basically if I have a super secret long and complicated password for a specific vault but want to access it with a pin (we can have the encrypted key inside the app memory just like the bitwarden password manager app when you have the the option to unlock with a pin but only on this specific device) so we can unlock the vault without having to enter a password but at the same time if I moved the encrypted folder to another device I would have to enter the password (because using the numeric keypad means encrypting files with a number and if someone brute forces the gocryptfs key 🗝️ they could bypass the encryption 🔓 in minutes)
of course this would be optional ☑️ and the user would be the one to choose between the password or having it under a pin.

**Comment by hardcore-sushi** [comment]
If I understand correctly, you want to be able to open volumes in the same way as by fingerprint authentication, but with a pin code instead of biometric authentication, right?

**Comment by creeper451** [comment]
Yes that's right 👍

**Comment by NearestWater262** [comment]
isnt this already supported? there is an option to use a pin code layout keyboard for typing password.

**Comment by creeper451** [comment]
Use but this will use the pin number as the actual password which could be brute forced easily what I,m asking for is using a strong password for the container itself while having the password stored in the appdata and be encrypted by a pin number (just like the fingerprint authentication) which would make opening containers frequently much easier than entering a long password every time without having to weak the encryption of the container

**Comment by NearestWater262** [comment]
@creeper451 i agree, this should definitely be added. im tired of typing my long password over and over heheh
